# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1730695.html
# Commit: 6d98d185a076
# Full Hash: 6d98d185a07684b3538dd0052cb14b67cba38568
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2021-11-05 09:34:21
# Regressor Bug: 1730695
# File Overlap Count: 1
# Description:
#   Bug 1730695 - Fix panic when casting large value to i32 rect r=gfx-reviewers,bradwerth,nical,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D128122
# ==============================================================================

diff -r bba94c79f3e1 -r 6d98d185a076 gfx/tests/crashtests/1730695.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1730695.html	Thu Nov 04 19:44:32 2021 +0000
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+  <script>
+    document.addEventListener("DOMContentLoaded", () => {
+      const canvas = document.createElementNS("http://www.w3.org/1999/xhtml", "canvas")
+      document.documentElement.appendChild(canvas)
+      const context = canvas.getContext("2d")
+      context.arcTo(4.883000064524857e+307, 1.546661088875107e+308, 2.559553941389527e+307, 1.0153516333341025e+308, 9.633835430115978e+307)
+      canvas.animate([{
+        "scale": "1090282421 55%"
+      }], 1000)
+    })
+    setTimeout('document.documentElement.className = ""', 300);
+  </script>
+</head>
+</html>
\ No newline at end of file